pytest_fixtures/component/leapp.py

```diff
@@ -0,0 +1,16 @@
+# LEAPP Fixtures
+import pytest
+
+
+@pytest.fixture(scope="function")
+def check_pre_upgrade(target_sat):
+    target_sat.execute('yum update')
+    target_sat.execute('reboot')
+    target_sat.execute('yum install leapp-upgrade')
+
+
+@pytest.fixture(scope="function")
+def verify_post_upgrade_state(target_sat):
+    os_version = target_sat.execute('cat /etc/redhat-release')
```

Comment:
we already have some helpers around getting system version information. Check the hosts.py file first